conf/40-escape

```diff
@@ -2,6 +2,7 @@
 # Make zsh exit on escape
 # © 2013 Axel Wagner and contributors (see also: LICENSE)
 function _shellex_exit {
+    rm $SHELLEX_SIZE_FILE
```

Comment:
needed - yes. At least by just removing this line, the SHELLEX_SIZE_FILE does not get deleted. I think the preexec does not get called when typing ESC.